src/hooks/usePreferences.js

```diff
@@ -0,0 +1,38 @@
+import { useCallback, useRef } from 'react'
+import { useHistory, useLocation } from 'react-router-dom'
+import { getPreferencesSearch } from '../Routes'
+
+/**
+ * Hook to interact with the preferences
+ * @returns {Array} first parameter open preferences handler, second parameter close preferences handler
```

Comment:
```suggestion
 * @returns {Array} [open preferences handler, close preferences handler, current preferences screen]
```

I _think_ this is fine for JSDoc... do you think it makes the return description clearer?

Reply:
yeah i think that is ok the way you suggested